crnlib/crn_vector.cpp

```diff
@@ -49,11 +45,7 @@ bool elemental_vector::increase_capacity(uint min_new_capacity, bool grow_hint,
         return false;
 
       char buf[256];
-#ifdef _MSC_VER
       sprintf_s(buf, sizeof(buf), "vector: crnlib_malloc() failed allocating %u bytes", (uint)desired_size);
-#else
-      sprintf(buf, "vector: crnlib_malloc() failed allocating %u bytes", (uint)desired_size);

```

Comment:
`snprintf` might be more portable since it's C99 instead of C11. But don't use it on MSVC since it's broken.

Reply:
Do you mean the provided non-MSVC `sprintf_s` is less portable than `snprintf`?

Reply:
Right. `snprintf` is in C99 and C++11 while `sprintf_s` is only in C11. I don't know if it would be a problem nowadays, but the pattern with the other ones was to use  the `_s` variant only in MSVC. But then reimplementing `vector` hardly makes sense at all unless you are targeting a platform with poor language support.

Reply:
Not using the provided non-Win32 `sprintf_s` questions the whole purpose of providing an alternate `sprintf_s` on non-Win32. Anyway we can think about improving that later. Priority is to fix Dæmon CI.

I've reverted to the initial `ifdef` code and used `snprintf` instead.